### DISCUSSION TITLE
Unable to re-verify as student

### DISCUSSION THREAD

**Discussion by pe200012** [community/community#76303] [GitHub Education]
### Hi everyone,

I signed up as student in 2020 and enjoyed Github student pack since then. Yesterday I noticed that my copilot complaining about not being able to verify my status and directed me to subscribe. I read the guide on Github Education and tried to solve the issue by re-verify the student status, and then was rejected after submitting with my school ID:
```
Your application cannot be reviewed until you fix the following:

    Please correct your GitHub billing information with your last name exactly as it appears in your academic affiliation document. If you have only a single legal name, enter it in both the first and last name fields.
    The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.
    Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
    Please correct your GitHub billing information with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.

```
But both my billing information and the image I uploaded should have met these requirements! Could you please help with this problem?

**Comment by davixmns** [comment]
1. GitHub Billing Information:

Ensure that your GitHub billing information, specifically your first and last name, matches exactly as it appears on your academic affiliation document. If you only have a single legal name, enter it in both the first and last name fields.

2. Academic Affiliation Document:

The image you uploaded should clearly display your school's complete name, not just the school logo. If the document is not in English, include a translation alongside the original document. If possible, use your device's camera to capture the document to avoid any image manipulation concerns.

3. GitHub Profile First Name:

Verify that your GitHub profile's first name matches exactly as it appears on your academic affiliation document. Avoid using variations of your name or nicknames.

After making these corrections, resubmit your verification request. If the issue persists, consider contacting GitHub Support for further assistance. They can provide more detailed guidance and investigate any potential technical issues.
